products/eks.md

```diff
@@ -102,19 +114,25 @@ releases:
 > conformant for a predictable experience.
 
 Amazon EKS guarantees support for at least four production-ready versions of Kubernetes at any given
-time. A Kubernetes version is fully supported on EKS for 14 months after first being available on
-Amazon EKS. This is true even if upstream Kubernetes is no longer supporting a version available on
-Amazon EKS.
+time. A Kubernetes version is fully supported on EKS for 14 months as standard support and for up to
+26 months as exteneded support after first being available on Amazon EKS. This is true even if
+upstream Kubernetes is no longer supporting a version available on Amazon EKS.
 
 You can subscribe to upgrade notices (sent approximately 12 months after the Kubernetes version was
 released on Amazon EKS) on your [Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/).
 The notice includes the end of support date, which is at least 60 days from the date of the notice.
```

Comment:
```suggestion
You can subscribe to upgrade notices on your [Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/). The notice includes the end of support date, which is at least 60 days from the date of the notice.
```

Because if I understood correctly this is no longer true as the GA date is now the Kubernetes GA date, not the EKS one.